cloudknot/aws/batch.py

```diff
@@ -302,7 +302,7 @@ def _create(self):  # pragma: nocover
         # unit testing would be expensive
         bucket = self.job_definition.output_bucket
         sse = get_s3_params().sse
-        pickled_input = cloudpickle.dumps(self.input)
+        pickled_input = cloudpickle.dumps(self.input, protocol=pickle.DEFAULT_PROTOCOL)
```

Comment:
Should this be as follows? 

```suggestion
        pickled_input = cloudpickle.dumps(self.input, protocol=cloudknot.DEFAULT_PICKLE_PROTOCOL)
```

With that variable defined in the library `__init__.py` or imported into it.

Reply:
Yes! It should. Fixed in most recent commit. Thanks!